client/components/guests.jsx

```diff
@@ -1,16 +1,89 @@
 import React from 'react';
 
+import GuestsDropdown from './guestsDropdown.jsx';
 import styles from '../styles/guests.css';
 
-const Guests = () => {
-  return (
-    <div>
-      <div id="guests-label">
-        <span className={styles.label}>Guests</span>
+class Guests extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showDropdown: false,
+      adultCount: 1,
+      childrenCount: 0,
+      guestCount: 1,
+      infantCount: 0,
+    };
+    this.toggleDropdown = this.toggleDropdown.bind(this);
+    this.hideDropdown = this.hideDropdown.bind(this);
+    this.incrementChildrenCount = this.incrementChildrenCount.bind(this);
```

Comment:
consider keeping a general order of function assignments such as increment - decrement, increment - decrement

Reply:
Good catch, I'll fix that